scout/server/blueprints/variants/templates/variants/components.html

```diff
@@ -86,4 +86,8 @@
   ">
     {{ variant.gnomad_frequency|human_decimal if variant.gnomad_frequency else '~' }}
   </div>
+  {% if variant.cosmic_ids %}
+  <br>
+  <span style="border:1px; border-style:solid; border-color:#000000; padding: 0em;"> <strong> C </strong></span>
```

Comment:
Hi @mikaell, you could create a nice badge using the bootstrap 4 standards, and associate to it a link to cosmic too. Look at #1862, which does just this.

Reply:
Ok! I'll have a look

Reply:
Changed to use Bootstraps build in badges instead.